README.md

```diff
@@ -22,8 +22,8 @@ $ helm repo add deis https://github.com/deis/charts
 $ helm up
 $ helm fetch deis/deis
 $ helm install deis
```

Comment:
why do we need `helm install deis --namespace=deis` if the chart's manifest files have `namespace: deis` and we have `deis-namespace.yaml` too?